docs/SetupDatabaseForAzure.md

```diff
@@ -48,3 +48,24 @@ You can choose Flexible or Single server deployment for Azure Database for MySQL
 Note:-
 
 * The [Storage auto-grow](https://docs.microsoft.com/en-gb/azure/mysql/concepts-pricing-tiers#storage-auto-grow) feature automatically increases storage without impacting the workload and is enabled by default in server configuration while creating an Azure Database for MySQL instance.
+
+
+## Azure Database for PostgreSQL
+
+In this section, you will create an Azure Database for PostgreSQL.
```

Comment:
```suggestion
```
Just repeating the section title is meaningless.
If we should say something more, let's do it.

Reply:
Thank you for the suggestion.
I have updated the section with additional information.